README.md

```diff
@@ -1,3 +1,76 @@
 # RCO-Soul
 
-RCO client connection to a Soul backend
+## Description
+
+The purpose of this project is to demonstrate how to run a React admin client using Soul as a REST API service. Soul is an open-source REST API wrapper that exposes an SQLite database to any client.
+
+## Running Soul and the React-admin app locally
+
+1. Install Soul on your local machine globally by running:
+
+   ```
+   npm i soul-cli -g
+   ```
+
+   NOTE: You might need admin access, so if you are on Mac or Linux, use `sudo`.
+
+2. Go to the root folder of the project and install the npm packages by running:
+
+   ```
+   npm install
+   ```
+
+3. Copy the environment variables from the `.env.sample` file and modify the variables by running:
+
+   ```
+   cp .env.sample .env
+   ```
+
+   - `PORT`: the port on which the Soul server runs.
+   - `VITE_API_URL`: the API URL of Soul, which should be `http://localhost:<PORT>/api/tables`.
+
```

Comment:
but @AbegaM - if the react-app is installed as an extension, we don't need to run `dev:client`, do we:?

Reply:
yes, but is it good to run the react admin client as an extension for development? i thought for development the users may need to start the soul server and the react admin server in a separate process, but i can make the development setup to run in one command by running react admin as an extension

Reply:
Could you include the instructions for both combinations please?

Reply:
ok sure let me add a new script to start the react-admin client as an extension and i will update the README, and the CORS issue is now resolved, if we don't pass anything to the `-c` flag when starting soul, it will allow CORS from all origins.